django_feature_flip/helpers.py

```diff
@@ -19,4 +19,23 @@ def __getattr__(self, name):
         return getattr(self.instance, name)
 
     def enabled(self, feature_name, actor=None):
-        return Feature.objects.get(name=feature_name).enabled(actor)
+        feature = Feature.objects.get(name=feature_name)
+        if feature.enabled(actor):
+            return True
+
+        if actor:
+            return self.feature_enabled_for_a_group_of_actor(feature, actor)
+
+        return False
```

Comment:
Shouldn't you check if actor is not None before using it in `if feature.enabled(actor)`? 

Also, if we plan on adding code climate I think it will complain about the multiple `return` statements.